server/plugin.go

```diff
@@ -468,7 +469,7 @@ func (p *Plugin) syncUsers() {
 					}
 				} else {
 					// Skip syncing of MS Teams guest user.
-					p.API.LogDebug("Skipping syncing of the guest user", "MMUserID", mmUser.Id, "TeamsUserID", msUser.ID)
+					p.API.LogDebug("Skipping syncing of the guest user", "TeamsUserID", msUser.ID)
```

Comment:
why this change ?

Reply:
this was causing panic in the code as the Mattermost user don't exist for this case

Reply:
I've seen this change in another PR. Should we pull it out from this one for clarity?

Reply:
this isn't present in the main branch currently that's why it is showing as a change in this PR. Also, the debug log we updated in this PR https://github.com/mattermost/mattermost-plugin-msteams-sync/pull/361, was incorrect, so I have reverted that change as well.

Reply:
synced the PR and updated the changes